### ISSUE TITLE
Exercise 13.6 wrong answer of the first question

### ISSUE THREAD

**Issue by ChaoMai** [Mooophy/Cpp-Primer#402]
The first question:
> What is a copy-assignment operator?

Original answer:
> The copy-assignment operator is function named `operator=`.

That's not right.

```cpp
// ...
HasPtr &operator=(const HasPtr &rhs); // copy-assignment operator
HasPtr &operator=(int a); // not a copy-assignment operator
// ...

```

* According to p443 in C++ Primer 5th (Chinese edition),  a copy-assignment operator should takes an argument of the same type as the class. 
* According to [copy_assignment](http://en.cppreference.com/w/cpp/language/copy_assignment) in cppreference, it should takes exactly one parameter of type `T`, `T&`, `const T&`, `volatile T&`, or `const volatile T&`.

**Comment by pezy** [comment]
Agree with you, could you please pull a request about this bug?

**Comment by ChaoMai** [comment]
fixed in Pull requests #403

**Comment by Mooophy** [comment]
@pezy 

Nice Guy~ :walking:
